### PR TITLE
Add necessary changes for PasswordReset migration

### PR DESCRIPTION
**Pull request by Palid**
Related: https://github.com/matrix-org/matrix-react-sdk/pull/6635

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by Palid** [comment]
Related: https://github.com/matrix-org/matrix-js-sdk/pull/1864

File: src/client.ts [hunk 1]
```diff
@@ -7992,9 +7992,9 @@ export class MatrixClient extends EventEmitter {
         limit?: number,
         batch?: string,
     ): Promise<{
-        rooms: ISpaceSummaryRoom[];
-        events: ISpaceSummaryEvent[];
-    }> {
+            rooms: ISpaceSummaryRoom[];
+            events: ISpaceSummaryEvent[];
+        }> {
```
Comment (SimonBrandner) [inline src/client.ts:7997]: The indentation seems a little weird here :thinking:

Comment (Palid) [reply src/client.ts:7997]: /me is setting up prettier while being very angry

Comment (SimonBrandner) [reply src/client.ts:7997]: Ah :grimacing:

Comment (Palid) [reply src/client.ts:7997]: It's due to eslint rules.

**Review by gsouquet — APPROVED** [review]

**Pull request merged**